src/index.cpp

```diff
@@ -561,7 +561,6 @@ void Index<T, TagT, LabelT>::load(const char *filename, uint32_t num_threads, ui
         if (file_exists(labels_to_medoids))
         {
             std::ifstream medoid_stream(labels_to_medoids);
-            assert(label_num_pts == data_file_num_pts);
```

Comment:
Why is this assert removed?

Reply:
This assert statement is there just above the   line if (file_exists(labels_to_medoids)) condition.
Its seems duplicate